pyproject.toml

```diff
@@ -1,14 +1,14 @@
 [tool.poetry]
 name = "saiph"
 # Also modify in saiph/__init__.py
-version = "1.5.4"
+version = "1.5.5"
 description = "A projection package"
 authors = ["Octopize <help@octopize.io>"]
 license = "Apache-2.0"
 
 [tool.poetry.dependencies]
 python = ">=3.9,<3.12"
-pandas = "^1.3"
+pandas = ">=1.5.3"
```

Comment:
This allows a package depending on saiph to work with `pandas` v2.

The rest of the code has been updated to work with pandas 2, and pandas 1.

Reply:
Moreover, this allows us to not break the API/core if we don't update to pandas straight away.